lib/waterline/error/WLError.js

```diff
@@ -16,128 +16,15 @@ var _ = require('lodash');
  * @param  {Object} properties
  * @constructor {WLError}
  */
-function WLError( properties ) {
-
-  // Call super constructor (Error)
-  WLError.super_.call(this);
-
-  // Fold defined properties into the new WLError instance.
-  properties = properties||{};
-  _.extend(this, properties);
-
-  // Generate stack trace
-  // (or use `originalError` if it is a true error instance)
-  if (_.isObject(this.originalError) && this.originalError instanceof Error) {
-    this._e = this.originalError;
-  }
-  else this._e = new Error();
-
-  // Doctor up a modified version of the stack trace called `rawStack`:
-  this.rawStack = (this._e.stack.replace(/^Error(\r|\n)*(\r|\n)*/, ''));
-
-
-  // Customize `details`:
-  // Try to dress up the wrapped "original" error as much as possible.
-  // @type {String} a detailed explanation of this error
-  if (typeof this.originalError === 'string') {
-    this.details = this.originalError;
-  }
-  // Run toString() on Errors:
-  else if ( this.originalError && util.isError(this.originalError) ) {
-    this.details = this.originalError.toString();
-  }
-  // But for other objects, use util.inspect()
-  else if (this.originalError) {
-    this.details = util.inspect(this.originalError);
```

Comment:
This is probably a dumb question: we pass around a lot of error objects - why don't we need this? `util.format` and `util.inspect` seem useful for formatting error objects - is something else that we do or use doing this under the hood?